tests/integration_test/data/test_configs/authorization/shell_commands.yml

```diff
@@ -14,7 +14,7 @@ tests:
         "admin_user_name": "admin@a.org"
         "result":
           "type": "admin_api_response"
-          "data": "audit.log daemon_pid.fl local log.txt pid.fl readme.txt startup transfer"
+          "data": "audit.log daemon_pid.fl error.log error_log.txt local log.txt pid.fl readme.txt startup transfer"
```

Comment:
Based on https://github.com/NVIDIA/NVFlare/pull/3057, error.log was renamed to error_log.txt. The master_template.yml still needs to be changed back right @nvkevlu?https://github.com/NVIDIA/NVFlare/blob/e55c523612fa9cab56e448d72520254305cd7573/nvflare/lighter/impl/master_template.yml#L329

Then we won't have both error.log and error_log.txt here.

Reply:
Correct, we should change master_template.yml to be error_log.txt instead of error.log so we won't need error.log here